src/cpp/rtps/security/SecurityPluginFactory.cpp

```diff
@@ -75,3 +75,20 @@ Cryptography* SecurityPluginFactory::create_cryptography_plugin(const PropertyPo
 
     return plugin;
 }
+
+Logging* SecurityPluginFactory::create_logging_plugin(const PropertyPolicy& property_policy)
+{
+    Logging* plugin = nullptr;
+    const std::string* logging_plugin_property = PropertyPolicyHelper::find_property(property_policy,
+            "dds.sec.log.plugin");
+
+    if(logging_plugin_property != nullptr)
```

Comment:
Elsewhere in the code there are spaces between the `if` and the `(`.

Reply:
But in this particular file there is none.

Reply:
Oh. Well then. :stuck_out_tongue: .